examples/vite-basic/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "vite-basic",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview"
+  },
+  "dependencies": {
+    "@vue/composition-api": "^1.0.0-rc.3",
+    "swrv": "^0.9.3",
```

```suggestion
    "swrv": "beta",
```

Thanks @bikingbadger there is already a vite example, but it's on vite `1.x` and on the `next` branch, but I like having a vite example on master. Can you update this to swrv beta?

---

Done